src/features/goblins/bank/components/WithdrawItems.tsx

```diff
@@ -97,65 +113,102 @@ export const WithdrawItems: React.FC<Props> = ({ onWithdraw }) => {
 
   return (
     <>
-      <span className="text-shadow text-base">Select items to withdraw</span>
-
-      <div className="flex flex-wrap h-fit">
-        {inventoryItems.map((itemName) => (
-          <Box
-            count={inventory[itemName]}
-            // isSelected={selected.includes(itemName)}
-            key={itemName}
-            onClick={() => onAdd(itemName)}
-            image={ITEM_DETAILS[itemName].image}
-            locked={
-              !canWithdraw({ item: itemName, game: goblinState.context.state })
-            }
-          />
-        ))}
-        {/* Pad with empty boxes */}
-        {inventoryItems.length < 4 &&
-          new Array(4 - inventoryItems.length)
-            .fill(null)
-            .map((_, index) => <Box disabled key={index} />)}
-      </div>
-
-      <div className="mt-2">
-        <span className="text-shadow text-base">Selected</span>
+      <div className="mt-3">
+        <div className="flex items-center border-2 rounded-md border-black p-2 bg-green-background mb-3">
```

Comment:
<img width="532" alt="Screen Shot 2022-05-07 at 8 53 35 am" src="https://user-images.githubusercontent.com/11745561/167225678-968afbcd-83e9-495e-ba8e-b370f7ff0a10.png">

I would suggest omitting the green text part and opting for using the timer icon on top of the icon you previously had. It should be clear to the players that an item takes X time to be crafted.

On my laptop, the modal height was very jarring and took up most of the page. The 3 warnings on this page was a bit overwhelming as well.

Reply:
Yep cool. I have changed the narrative. I would argue that the green sign stays. I don't think it is immediately clear as to why you can't withdraw some items. Especially the ones that are "in use". Its not immediately clear that a scarecrow is in use just because crops are planted. It would also be easy to miss that something is still in the 7 day cooldown period. People may not pay attention to that and then you come here and all the sudden they're disabled. It doesn't give a lot of feedback.

I have removed the second red warning. See what you think. If you still aren't convinced I will remove the green.